lib/file_validators/validators/file_content_type_validator.rb

```diff
@@ -22,6 +22,8 @@ def validate_each(record, attribute, value)
           validate_whitelist(record, attribute, content_type, allowed_types)
           validate_blacklist(record, attribute, content_type, forbidden_types)
         end
+      rescue JSON::ParserError
+        record.errors.add attribute, :invalid
```

Comment:
Is `:invalid` a documented active model error message?

Reply:
It's part of [the default set of error message embedded in the active_model gem](https://github.com/rails/rails/blob/master/activemodel/lib/active_model/locale/en.yml#L12)

Reply:
:+1: